.circleci/config.yml

```diff
@@ -46,7 +46,7 @@ commands:
       coverage:
         type: boolean
         default: false
-      compat:
```

Comment:
I decided the word `compat` was too overloaded at this point, so I renamed this to `legacy_browsers`.